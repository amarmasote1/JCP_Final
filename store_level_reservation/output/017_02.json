{
    "warnings": [],
    "message": "Successful",
    "nodeType": null,
    "body": null,
    "key": "JCP|0647",
    "status": 221
}