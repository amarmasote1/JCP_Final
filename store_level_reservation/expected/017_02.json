{"status":221,"message":"Successful","key":"JCP|0647","body":null,"warnings":[]}